english/coffee/cookie/bake-cookies.md

```diff
@@ -5,3 +5,8 @@ You satiate yourself with a pan
 full of freshly baked cookies.
 
 Yum.
+
+After having coffee and cookies, you would want to:
+
+[Go for a walk](walk/walk.md)
+[Go for a swim](swim/swim.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.